src/modules/newtek/interop/Processing.NDI.FrameSync.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+//***********************************************************************************************************************************************
+// 
+// Copyright(c) 2014-2018 NewTek, inc
```

Comment:
What is the copyright license of this file?  Without any extra information, like the one in Processing.NDI.DynamicLoad.h, the source code appear to be incompatible with the GPL.

Reply:
The FrameSync thing seems to be a work in progress, looking at the amount of typos in comments, and it not being included in the NDIlib_v3 struct. I'll try to contact NewTek to get some clarification.

Reply:
@ianshade I already did contact them and got a response.
![image](https://user-images.githubusercontent.com/1327476/49480190-a1f92580-f81d-11e8-8d41-99d7acdf6a0c.png)